meta/runtime.yml

```
@@ -156,3 +168,7 @@ plugin_routing:
   terminal:
     routeros:
       redirect: community.routeros.routeros
+doc_fragments:
+  nso:
+    redirect: cisco.nso.nso
```

This is one level too far up:
```suggestion
  doc_fragments:
    nso:
      redirect: cisco.nso.nso
```